src/excmds.ts

```diff
@@ -1565,6 +1565,8 @@ export async function credits() {
  *
  * Suggested usage: `autocmd DocLoad .* no_mouse_mode`
  *
+ * To revert, use [[mouse_mode]]
```

Comment:
I think this implies that it removes the autocmd; we should be careful to make the distinction clear.